DebugConnection.py

```diff
@@ -24,13 +26,15 @@ def _run(self) -> None:
 
         """
         while True:
-            time.sleep(1)
+            time.sleep(5)
             with self.lock:
                 if not self.callback:
                     continue
 
-                bulk_sensor_arr: bytearray = self.bulk_sensor_mock_random()
-                self.callback(bulk_sensor_arr)
+                # bulk_sensor_arr: bytearray = self.bulk_sensor_mock_random()
+                # statusPingArr: bytearray = self.statusPingMockSetValues()
+                messageArr: bytearray = self.messageValues()
+                self.callback(messageArr)
```

Comment:
DebugConnection doesn't seem to work anymore. All of the sensor values just become None (or stay 0) after ~5 seconds.

Reply:
Good catch! Manual CI lol

Reply:
I've been using it for testing and I'm not sure how's best to get all the info.
Let's discuss this.